src/simoc_sam/displays/ssd1306.py

```diff
@@ -0,0 +1,102 @@
+import json
+import time
+import socket
+
+from PIL import Image, ImageDraw, ImageFont
+
+from ..sensors import utils
+from .. import config
+
+board = utils.import_board()
+import digitalio
+import adafruit_ssd1306
+
+
+WIDTH, HEIGHT = 128, 64
+oled_reset = digitalio.DigitalInOut(board.D4)
+oled = adafruit_ssd1306.SSD1306_I2C(
+    WIDTH, HEIGHT, board.I2C(), addr=0x3D, reset=oled_reset
+)
+
+# Fonts
+font_small = ImageFont.load_default()
+font_header = ImageFont.truetype(
+    "/usr/share/fonts/truetype/dejavu/DejaVuSans-Bold.ttf", 14
+)
+
+# Record system start time for uptime
+START_TIME = time.monotonic()
+
+def read_latest_entry(filepath):
+    try:
+        with open(filepath) as f:
+            for line in f:
+                pass
+            if line.strip():
+                return json.loads(line)
+    except Exception as err:
+        print(err)
+    return {}
+
+def format_values(data_scd, data_bme, data_bno):
+    co2 = data_scd.get("co2", "--")
+    temp = data_scd.get("temperature", "--")
+    hum = data_scd.get("humidity", "--")
+    pres = data_bme.get("pressure", "--")
+    accx = data_bno.get("accel_x", "--")
+    accy = data_bno.get("accel_y", "--")
+    accz = data_bno.get("accel_z", "--")
+    return [
+        "CO2: " + f"{co2:.0f}" if isinstance(co2, float) else str(co2),
+        "Tmp: " + f"{temp:.1f}" if isinstance(temp, float) else str(temp),
+        "Hum: " + f"{hum:.1f}" if isinstance(hum, float) else str(hum),
+        "Prs: " + f"{pres:.1f}" if isinstance(pres, float) else str(pres),
+        "A-x: " + f"{accx:.2f}" if isinstance(accx, float) else str(accx),
+        "A-y: " + f"{accy:.2f}" if isinstance(accy, float) else str(accy),
+        "A-z: " + f"{accz:.2f}" if isinstance(accz, float) else str(accz),
+    ]
+
+def format_uptime():
+    t = int(time.monotonic() - START_TIME)
```

Comment:
```suggestion
    t = int(time.monotonic())
```

Calculating the diff with the `START_TIME` gives the script uptime, not the RPi uptime.